_content/conf.py

```diff
@@ -13,15 +13,15 @@
 import os
 import sys
 sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(1, '/usr/local/lib/python3.7/site-packages/')
+sys.path.insert(1, '/opt/homebrew/lib/python3.9/site-packages')
 import recommonmark
 from recommonmark.transform import AutoStructify
 
 
 # -- Project information -----------------------------------------------------
 
 project = 'Agora Developer Document Style Guide'
-copyright = '2020, Agora.io'
+copyright = '2021, Agora.io'
```

Comment:
Company is called "Agora, Inc." on NASDAQ. Web site is "Copyright 2021 Agora". Is "Agora.io" a legitimate form of the company name?